### ISSUE TITLE
ome-zarr mobie samples

### ISSUE THREAD

**Issue by constantinpape** [mobie/mobie.github.io#98]
Here is an overview of current MoBIE projects, which have ome.zarr data and samples:

## Spatial Transcriptomics Data

https://github.com/mobie/mouse-embryo-spatial-transcriptomics-project

This contains 40 or so positions, each with image data (stored as ome.zarr on the ebi-s3) and transcriptomics data stored as csv.
The data for the first position is available here:
- `image data`: https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0138A/TimEmbryos-120919/HybCycle_29/MMStack_Pos0.ome.zarr
- `cell segmentation`: https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0138A/TimEmbryos-120919/HybCycle_29/MMStack_Pos0.ome.zarr/labels/cells

The image data has 4 channels.

## Covid HTM Data

https://github.com/mobie/covid-if-project

This contains data from a HTM screen for covid antibodies and contains a few hundred image positions.
Here are the addresses for one of the positions:
- nuclei: https://s3.embl.de/covid-if-project/20200406_164555_328/images/ome-zarr/nuclei_WellE06_0000.ome.zarr
- serumIgG: https://s3.embl.de/covid-if-project/20200406_164555_328/images/ome-zarr/serumIgG_WellE06_0000.ome.zarr
- marker: https://s3.embl.de/covid-if-project/20200406_164555_328/images/ome-zarr/marker_tophat_WellE06_0000.ome.zarr
- cell_segmentation: https://s3.embl.de/covid-if-project/20200406_164555_328/images/ome-zarr/cell_segmentation_WellE06_0000.ome.zarr
- nucleus_segmentation: https://s3.embl.de/covid-if-project/20200406_164555_328/images/ome-zarr/nucleus_segmentation_WellE06_0000.ome.zarr

## Platynereis Data

The raw data and cell segmentation from https://github.com/mobie/platybrowser-project is also available as ome.zarr (the data for the project itself is stored as n5):
- raw:  https://s3.embl.de/i2k-2020/platy-raw.ome.zarr
- cell-segmentation:  https://s3.embl.de/i2k-2020/platy-raw.ome.zarr/labels/cells

**Comment by will-moore** [comment]
Hi @constantinpape - I tried to open a few of those Covid HTM samples in ome-ngff-validator, and in neuroglancer but I'm getting CORs errors. E.g. https://ome.github.io/ome-ngff-validator/?source=https://s3.embl.de/covid-if-project/20200406_164555_328/images/ome-zarr/nuclei_WellE06_0000.ome.zarr
The Spatial Transcriptomics are working 👍

**Comment by constantinpape** [comment]
> I tried to open a few of those Covid HTM samples in ome-ngff-validator, and in neuroglancer but I'm getting CORs errors.

Yes, some option is not enabled by default on the EMBL S3 and we have run into this issue before. @tischi do you remember how this was fixed by Josep? I think you also solved this with him recently.

**Comment by tischi** [comment]
Which bucket? 
What is the exact error message?
I think I have to ask Josep to fix this.

**Comment by tischi** [comment]
@will-moore could you please try again?

**Comment by will-moore** [comment]
Looks good 👍 thanks.
However, the validator is reporting `Dimension separator must be / for version 0.4`.
I believe this will be relaxed in future versions (https://github.com/ome/ngff/pull/161) but currently that won't be read by ome-zarr-py (see fix in https://github.com/ome/ome-zarr-py/pull/235)

**Comment by tischi** [comment]
Wow, I am surprised that this data is so ancient that we still have `.` separator...

**Comment by tischi** [comment]
@constantinpape once you have more time again, maybe good to double check this.

**Comment by constantinpape** [comment]
Yes, I can confirm: 
```
$ mc ls embl/covid-if-project/20200406_164555_328/images/ome-zarr/nuclei_WellE06_0000.ome.zarr/s0
[2021-11-19 14:35:32 CET]   271B STANDARD .zarray
[2021-11-19 14:35:32 CET]    31B STANDARD .zattrs
[2021-11-19 14:35:32 CET] 856KiB STANDARD 0.0
[2021-11-19 14:35:32 CET] 859KiB STANDARD 0.1
[2021-11-19 14:35:32 CET] 705KiB STANDARD 1.0
[2021-11-19 14:35:32 CET] 700KiB STANDARD 1.1
```

I think that stems from a bug when creating this data that has since been fixed. (And technically the current data is not valid; it specifies format version 0.4, but uses .). -> This should be fixed and it's in principle easy to write something that does that on the fly, but I don't have time for that right now, maybe next week.

**Comment by titansmc** [comment]
The CORS error is gone now, we had one migration done and I have dissallowed the ome.github.io domain. It is back now.
Cheers.
